### DISCUSSION TITLE
Shared XP and gold for multiplayer.

### DISCUSSION THREAD

**Discussion by Azurzar** [diasurgical/DevilutionX#7050] [Q&A]
### Feature Type

Quality of Life

### Describe

Hi! Is there any way to have this (I think prity elemental) features?
Ty!

**Comment by Trihedraf** [comment]
They are not currently implemented. 
There is a draft PR for sharing xp #6293

**Comment by StephenCWills** [comment]
It seems reasonable to assume that these would represent prime examples of features that could be added as mods once the Lua implementation has matured.

**Comment by AJenbo** [comment]
As long as both players take part in killing a monster they will both get xp from it

**Comment by kphoenix137** [comment]
> ### Feature Type
> 
> Quality of Life
> 
> ### Describe
> 
> Hi! Is there any way to have this (I think prity elemental) features?
> Ty!

Shared gold? What does that mean. You can already share it

**Reply by Trihedraf** [reply]
In Diablo 3 when you pickup gold everybody gets it.
